src/navigation/Router.tsx

```diff
@@ -72,22 +73,25 @@ function Router() {
             </PrivateRoute>
           }
         />
+        {/* Pagina di Modifica/Dettaglio aggregato*/}
         <Route
           path={routes.UPDATE_AGGREGATE}
           element={
-            <PrivateRoute roles={[Permission.API_KEY_WRITE]}>
+            <PrivateRoute roles={[Permission.API_KEY_READ]}>
               <AggregateDetailPage />
             </PrivateRoute>
           }
         />
+        {/* Pagina di Creazione Aggregato*/}
         <Route
           path={routes.AGGREGATE}
```

Comment:
Please change the name `AGGREGATE` to `CREATE_AGGREGATE`, to avoid confusions about the intent of this route.